src/main/java/austin/Task.java

```diff
@@ -0,0 +1,43 @@
+package austin;
+
+public abstract class Task {
+    /** Description of the task */
+    protected String description;
+
+    /** Indicates whether the task is done */
+    protected boolean isDone;
+
+    public Task(String description) {
+        setDescription(description);
+        setDone(false);
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public void setDone(boolean done) {
```

Comment:
Maybe it is more appropriate to have the boolean argument's name as isDone as well.